sdk/freeport/freeport.go

```diff
@@ -381,11 +394,44 @@ func logf(severity string, format string, a ...interface{}) {
 	fmt.Fprintf(os.Stderr, "["+severity+"] freeport: "+format+"\n", a...)
 }
 
+// TestingT is the minimal set of methods implemented by *testing.T that are
+// used by functions in freelist.
+//
+// In the future new methods may be added to this interface, but those methods
+// should always be implemented by *testing.T
+type TestingT interface {
+	Helper()
+	Fatalf(format string, args ...interface{})
+	Cleanup(func())
+}
+
+// GetN returns n free ports from the reserved port block, and returns the
+// ports to the pool when the test ends. See Take for more details.
+func GetN(t TestingT, n int) []int {
```

Comment:
The naming is becoming a bit confusing, what about `PortN` so we have all the higher level API (the one that take a `TestingT`) named the same?

Reply:
Ya, I'm happy to change the name of this.  I wonder if `PortN` could imply "reserve this TCP port for me". 

We could also change `Port` to `Get1(t *testing.T)` which would be more consistent.

What do you think? I'm happy to keep thinking about more options.

Reply:
I agree that `PortN` might be ambiguous.

I like `GetOne` and `GetN` since they play nicely with the package name:
```
freeport.GetOne
freeport.GetN
```